backend/FwLite/LcmCrdt/CrdtMiniLcmApi.cs

```diff
@@ -388,9 +388,16 @@ private async IAsyncEnumerable<Entry> GetEntries(
             .LoadWith(e => e.Senses).ThenLoad(s => s.PartOfSpeech)
             .LoadWith(e => e.ComplexForms)
             .LoadWith(e => e.Components)
-            .AsQueryable()
-            .OrderBy(e => e.Headword(sortWs.WsId).CollateUnicode(sortWs))
-            .ThenBy(e => e.Id);
+            .AsQueryable();
+        if (options.Order.Ascending)
+        {
+            queryable = queryable.OrderBy(e => e.Headword(sortWs.WsId).CollateUnicode(sortWs)).ThenBy(e => e.Id);
+        }
+        else
+        {
+            queryable = queryable.OrderByDescending(e => e.Headword(sortWs.WsId).CollateUnicode(sortWs)).ThenBy(e => e.Id);
```

Comment:
This seems correct to me. So, the list actually "flips", but...it's not very important.
```suggestion
            queryable = queryable.OrderByDescending(e => e.Headword(sortWs.WsId).CollateUnicode(sortWs)).ThenByDescending(e => e.Id);
```